README.md

```diff
@@ -118,6 +96,39 @@ class {'kubernetes':
 }
 ```
 
+### Validating and unit testing the module
+
+This module is compliant with the Puppet Development Kit [(PDK)](https://puppet.com/docs/pdk/1.x/pdk.html), which provides tools to help run unit tests on the module and validate the modules's metadata, syntax, and style. 
+
+*Note:* To run static validations and
+unit tests against this module using the [`pdk validate`](https://puppet.com/docs/pdk/1.x/pdk_reference.html#pdk-validate-command) and [`pdk test unit`](https://puppet.com/docs/pdk/1.x/pdk_reference.html#pdk-test-unit-command) commands, you must have Puppet 5 or higher installed. In the following examples we have specified Puppet 5.3.6.
+
+To validate the metadata.json file, run the following command:
+
+```puppet
+pdk validate metadata --puppet-version='5.3.6'
+```
+
+To validate the Puppet code and syntax, run the following command:
+
+```puppet
```

Comment:
These technically aren't puppet code, they are CLI commands. Aside from that the commands themselves and the descriptions look grand :)

Reply:
Thanks @HelenCampbell